packages/react-components/react-dialog/src/components/DialogTitle/renderDialogTitle.tsx

```diff
@@ -1,12 +1,19 @@
-import * as React from 'react';
-import { getSlots } from '@fluentui/react-utilities';
+/* eslint-disable jsdoc/check-tag-names */
+/** @jsxRuntime classic */
+/** @jsxFrag Fragment */
```

Comment:
This might be not intuitive as we use `Fragment` only sometimes, example: I copied comments and an import from `useDialogTitle.tsx`, added a fragment, nothing works 🥲

Reply:
> nothing works 🥲

You'll be receiving clear error messages about this on build time and also on most code editors, I don't see why this is a problem. This would have a very big red flag on those specific cases, in any way... what would be the alternative?

Reply:
As an alternative, we can keep React's import and avoid using `jsxFrag`. In this case, `Fragment` will be imported from React's package.

Not sure, what is more intuitive, so feel free to decide what works better.